django/contrib/postgres/fields/ranges.py

```diff
@@ -132,6 +133,44 @@ def db_type(self, connection):
 RangeField.register_lookup(lookups.Overlap)
 
 
+class DateTimeRangeContains(models.Lookup):
```

Comment:
The proposed text makes sense but leaves me wondering where I can read more about this limitation and if it might be addressed someday.